packages/material-ui/src/Slider/Slider.js

```diff
@@ -247,11 +247,12 @@ export const styles = theme => ({
     '&::after': {
       position: 'absolute',
       content: '""',
-      // reach 48px touch target (2 * 18 + thumb circumference)
-      left: -18,
-      top: -18,
-      right: -18,
-      bottom: -18,
+      borderRadius: '50%',
+      // x10 thumb surface (from 12px^2 to 40px^2).
+      left: -14,
```

Comment:
Shouldn't this be -16? If we had 48px before and want to reduce it to 44px we only need to reduce it by 2px on both sides which results in a 2*2px reduction.